### PR TITLE
rm speedy ab test

### PR DESCRIPTION
**Pull request by prattprattpratt**

### CODE REVIEW

File: scripts/config/templates/nginx.conf.erb [hunk 1]
```diff
@@ -138,76 +111,15 @@ http {
 
   rewrite ^/embed/(.*)$ /$1.html last;
 
-  # the speedy_search_assignment cookie forces nginx to always send that user to that arm of the test
-  # the speedy_should_identify cookie tells the clients to identify the assignment in Rudderstack
-  # 1. Speedy Search - SSR
-  if ($speedy_search_condition = "on_<%= ssr_frontend_host %>_/search") {
-    rewrite ^(.*)$ /speedy-search/$1;
+  location ^~ /search {
+    proxy_pass <%= ssr_frontend_host %>;
```
Comment (calengbuford) [inline scripts/config/templates/nginx.conf.erb:115]: Did you test this locally at all?

File: scripts/config/templates/nginx.conf.erb [hunk 2]
```diff
@@ -138,76 +111,15 @@ http {
...
+
   # redirect to speedy home page in ssr-frontend
   location = / {
     proxy_pass <%= ssr_frontend_host %>/$is_args$args;
   }
 
-  location ^~ /speedy-search {
-    rewrite ^/speedy-search/(.*)$ $1 break;
-    add_header Set-Cookie "speedy_search_assignment=<%= ssr_frontend_host %>; Max-Age=31536000; Path=/";
-    add_header Set-Cookie "speedy_should_identify=$speedy_should_identify_default_true; Max-Age=31536000; Path=/";
```
Comment (calengbuford) [inline scripts/config/templates/nginx.conf.erb:156]: We'll need to remove this check in SSR so don't stop identify events.

File: scripts/config/templates/nginx.conf.erb [hunk 3]
```diff
@@ -138,76 +111,15 @@ http {
...
-    add_header x-speedy-redir <%= ssr_frontend_host %>;
-
-    proxy_pass <%= ssr_frontend_host %>;
-  }
-
-  # redirect /search-q, /speedy-search/count, and /speedy-search/{listingId} to ssr-frontend
-  location ~ ^/search((-q)|/?(count|\d+))$ {
```
Comment (calengbuford) [inline scripts/config/templates/nginx.conf.erb:163]: Let's make sure these still work

File: scripts/config/templates/nginx.conf.erb [hunk 4]
```diff
@@ -138,76 +111,15 @@ http {
...
-    add_header Set-Cookie "speedy_search_assignment=<%= ssr_frontend_host %>; Max-Age=31536000; Path=/";
-    add_header Set-Cookie "speedy_should_identify=$speedy_should_identify_default_false; Max-Age=31536000; Path=/";
```
Comment (calengbuford) [inline scripts/config/templates/nginx.conf.erb:165]: Why didn't we identify here? @brundagejoe

Comment (calengbuford) [reply scripts/config/templates/nginx.conf.erb:165]: I don't remember why we did this.

File: scripts/config/templates/nginx.conf.erb (hunk 2 above)
Comment (prattprattpratt) [reply scripts/config/templates/nginx.conf.erb:156]: Isn't this just for the `identify`s for the A/B test user property `AB_useSpeedySearch`

File: scripts/config/templates/nginx.conf.erb (hunk 1 above)
Comment (prattprattpratt) [reply scripts/config/templates/nginx.conf.erb:115]: I'm testing it now

File: scripts/config/templates/nginx.conf.erb (hunk 2 above)
Comment (calengbuford) [reply scripts/config/templates/nginx.conf.erb:156]: You're right

**Review by calengbuford — APPROVED** [review]

**Pull request merged**